bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceScreen.kt

```diff
@@ -0,0 +1,213 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import android.widget.TextView
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.hasChildCount
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withClassName
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Matchers.`is`
+import org.junit.Assert.assertNotEquals
+
+class DigitalInvoiceScreen {
+    private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    private var initialValue: String? = null
+    private var updatedValue: String? = null
+
+
+    fun displayDigitalInvoiceTextOnOnboardingScreen(): DigitalInvoiceScreen {
+        val onboardingScreenText = device.findObject(
+            UiSelector()
+                .className("android.widget.TextView")
+                .text("Digital invoice")
+                .resourceId("net.gini.android.bank.sdk.exampleapp:id/onboarding_text_1")
+        )
+        onboardingScreenText.exists()
+        return this
+    }
+
+    fun displayGetStartedButtonOnOnboardingScreen(): DigitalInvoiceScreen {
+        val onboardingScreenButton = device.findObject(
+            UiSelector()
+                .className("android.widget.Button")
+                .text("Get Started")
+                .resourceId("net.gini.android.bank.sdk.exampleapp:id/done_button")
+        )
+        onboardingScreenButton.exists()
+        return this
+    }
+
+    fun clickGetStartedButtonOnOnboardingScreen(): DigitalInvoiceScreen {
+        val onboardingScreenButton = device.findObject(
+            UiSelector()
+                .className("android.widget.Button")
+                .text("Get Started")
+        )
+        onboardingScreenButton.exists()
+        onboardingScreenButton.click()
+        return this
+    }
+
+    fun assertDigitalInvoiceText(): DigitalInvoiceScreen {
```

Comment:
I can remove `DigitalInvoiceScreen`..

But I don't see any need to put a condition as the method is simply asserting the existence of `Digital Invoice ` text..

Reply:
The reason why I am saying this is because if the assertion is not returned, the main test case will pass either way.

For example, if the method is empty `fun assertDigitalInvoiceText() {  }` , then the main test case will pass, because it does not use the result from the assertion. 

However, if we have something like `fun assertDigitalInvoiceText(): Boolean { 
return digitalInvoiceText.exists()
 }`, then in the main test case we can use it like so: `val isDigitalInvoiceTextDisplayed = assertDigitalInvoiceText()
assertThat(true, isDigitalInvoiceTextDisplayed)
`

This way in the first method we return if it is displayed or not, and in the test case we check if it is displayed. Otherwise even if the text is not displayed, the test case will pass.